WultraMobileTokenSDK/Operations/Model/UserOperation/Attributes/WMTOperationAttributeAmount.swift

```diff
@@ -39,18 +39,25 @@ public class WMTOperationAttributeAmount: WMTOperationAttribute {
     ///
     /// For example when the currency is CZK, this property will be "Kč"
     public let currencyFormatted: String?
+
+    /// Formatted value and currency to the locale based on acceptLanguage
+    ///
+    /// Both amount and currency are formatted, String will show e.g. "€" in front of the amount
+    /// or "EUR" behind the amount depending on locale
```

Comment:
On __the__ locale. I'm recommending using Grammarly or something similar to catch such errors